src/components/Avatar/Avatar.tsx

```diff
@@ -48,33 +47,33 @@ export interface AvatarProps extends VibeComponentProps {
 
 const Avatar: React.FC<AvatarProps> = ({
   id,
-  type,
+  type = AvatarTypes.TEXT,
   className,
-  textClassName,
-  size,
+  textClassName = "",
+  size = AvatarSizes.LARGE,
```

Comment:
It's customary to have enum names as plural, not singular, same for `AvatarTypes`.